scipy/stats/_multivariate.py

```diff
@@ -151,10 +155,13 @@ class _PSD:
     """
 
     def __init__(self, M, cond=None, rcond=None, lower=True,
-                 check_finite=True, allow_singular=True):
+                 check_finite=True, allow_singular=True, check_symmetric=True):
```

Comment:
I would keep it `False` to keep the performance same as before this change. Not sure how much affect does this check has on execution time.